twisted/twisted/plugins/tunnel_helper_plugin.py

```diff
@@ -53,7 +124,6 @@ def start_walking(self):
 
 
 class Tunnel(object):
-
     __single = None
```

Comment:
I feel like this whole class should be elsewhere. It's something that can be used outside of the twistd plugin.